treq/api.py

```diff
@@ -110,5 +114,11 @@ def _client(*args, **kwargs):
     pool = default_pool(reactor,
                         kwargs.get('pool'),
                         kwargs.get('persistent'))
-    agent = Agent(reactor, pool=pool)
+    proxy = kwargs.get('proxy')
+    if proxy is not None:
+        (address, port) = proxy
+        endpoint = TCP4ClientEndpoint(reactor, address, port)
```

Comment:
Hm, this is IPv4-only…I’d find that unfortunate for a new API.

I understand treq tries to imitate requests’s API, maybe we should do something similar to http://docs.python-requests.org/en/latest/user/advanced/#proxies ?